src/core/src/lib/models/fieldconfig.ts

```diff
@@ -5,7 +5,7 @@ import { FieldWrapper } from '../templates/field.wrapper';
 import { ValidationMessageOption } from '../models';
 import { Type } from '@angular/core';
 
-export interface FormlyFieldConfig {
+export interface FormlyFieldConfig<Props = FormlyFieldProps & { [additionalProperties: string]: any }> {
```

Comment:
```suggestion
export interface FormlyFieldConfig<Props = FormlyFieldProps> {
```

Why does this again have the additionalProperties? I thought this should only be allowed on templateOptions.

Reply:
`FormlyFieldConfig` should allow additionalProperties, to make it strict simply use `FormlyFieldConfig<FormlyFieldProps>`. You may check #3303 to have a clear idea for `strict` mode.